### DISCUSSION TITLE
ALPINE_VERSION mismatch

### DISCUSSION THREAD

**Discussion by rrpadilla** [alpinejs/alpine#1691] [7. Bugs (archived)]
Seems like the **build** method is not getting the right ALPINE_VERSION [from here](https://github.com/alpinejs/alpine/blob/v3.2.1/scripts/build.js#L78-L89).

The latest version **3.2.1** is showing **version: "3.2.0"** instead of **version: "3.2.1".**
```script
Alpine.version    // returns "3.2.0"
```

[https://unpkg.com/alpinejs@3.2.1/dist/cdn.min.js](https://unpkg.com/alpinejs@3.2.1/dist/cdn.min.js)
[https://unpkg.com/alpinejs@3.2.1/dist/cdn.js](https://unpkg.com/alpinejs@3.2.1/dist/cdn.js)

**Comment by james0r** [comment]
Same on npm package.
